sn_protocol/src/error.rs

```diff
@@ -37,6 +37,8 @@ pub enum Error {
     InvalidRegister(RegisterAddress),
     #[error("Register is Invalid: {0}")]
     RegisterError(#[from] sn_registers::Error),
+    #[error("The Register was already created by another owner: {0:?}")]
+    RegisterAlreadyClaimed(bls::PublicKey),
```

Comment:
better using Register::name or Register::address to be consistent with other logging fmt

Reply:
Agree that we need consistency, but in this context, I think it might be more interesting to know who already created the register rather than the address.